requirements.txt

```diff
@@ -1,2 +1,2 @@
-PyYAML
-requests
\ No newline at end of file
+requests==2.19.1
+PyYAML==4.1
```

Comment:
When I tried to install packages via `pip3`, I ran into the following issue:

```
$ pip3 install -r requirements.txt

Collecting requests==2.19.1 (from -r requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/65/47/7e02164a2a3db50ed6d8a6ab1d6d60b69c4c3fdf57a284257925dfc12bda/requests-2.19.1-py2.py3-none-any.whl (91kB)
    100% |████████████████████████████████| 92kB 3.1MB/s
Collecting PyYAML==4.1 (from -r requirements.txt (line 2))
  Could not find a version that satisfies the requirement PyYAML==4.1 (from -r requirements.txt (line 2)) (from versions: 3.10, 3.11, 3.12)
No matching distribution found for PyYAML==4.1 (from -r requirements.txt (line 2))
```

How did you generate `requirements.txt` file? You could create a virtual environment with [virtualenv](https://virtualenv.pypa.io/en/stable/) (make sure adding ignore rules in you .gitignore, e.g. https://www.gitignore.io/api/python), and generate the `requirements.txt` file with this command:

```
$ pip3 freeze > requirements.txt
```

Reply:
I generated `requirements.txt` with `pipreqs`. It looks like I had PyYAML 4.1 installed for some reason but after reinstalling and running `pipreqs` again, the version is 3.12 which works when using `pip3 install -r requirements.txt`. I tried using `virtualenv` and `pip3 freeze` but it generated a list of ~26 dependencies. Should I use this method over `pipreqs`?